ui/pages/send/gas-display/index.scss

```diff
@@ -47,7 +47,8 @@
         padding: 0 32px 16px 16px;
         position: fixed;
         bottom: 80px;
-        z-index: 1;
+        // Set z-index above ".send-v2__hex-data__input". This should be a temp fix. TODO: Refactor z-indexes
+        z-index: 1026;
```

Comment:
@digiwand : curious why you preferred to use 1026 ?

Reply:
hm. I wrote the reason in the description of this PR but maybe I should move more of it to this comment

Reply:
@jpuri updated https://github.com/MetaMask/metamask-extension/pull/17470/commits/dfb99411752d8eeeb13a43b2949e6ce1df717d3b ✌️